lib/npm/install.js

```diff
@@ -38,24 +41,41 @@ function install(context, next) {
     return next(Error('Install Timed Out'));
   }
 
+  var installOutput = '';
+  var installError = '';
+
   proc.stderr.on('data', function (chunk) {
     context.emit('data', 'warn', 'npm-install:', chunk.toString());
+    if (context.module.stripAnsi) {
+      chunk = stripAnsi(chunk.toString());
+      chunk = chunk.replace(/\r/g, '\n');
+    }
+    installError += chunk;
   });
 
   proc.stdout.on('data', function (chunk) {
     context.emit('data', 'verbose', 'npm-install:', chunk.toString());
+    if (context.module.stripAnsi) {
+      chunk = stripAnsi(chunk.toString());
+      chunk = chunk.replace(/\r/g, '\n');
+    }
+    installOutput += chunk;
   });
 
   proc.on('error', function() {
     bailed = true;
     clearTimeout(timeout);
+    context.testOutput += installOutput;
+    context.testError += installError;
```

Comment:
if this doesn't exist this will result in an unwanted `undefined` at the beginning of the string